src/main/java/gov/usds/case_issues/config/DemoUserLoginConfig.java

```diff
@@ -25,25 +22,6 @@
 
 	@Autowired
 	private WebConfigurationProperties _webProperties;
-	@Autowired
-	private ApiInfo _apiInfo;
-
-	@Bean
-	@Order(-1)
-	public WebSecurityPlugin addDemoLogins() {
```

Comment:
I am torn on how we should be wiring these things up in dev versus deployed environments—it seems like we should be able to preserve the convenient form login for dev and have the oauth stuff work in deployed environments, and eat our 🍰 and all that good stuff. Probably we have to revisit this post launch and see how much effort it's worth? The `WebSecurityPlug` trick was intended to allow us to do this kind of thing flexibly, though, so I'm hopeful that we can do it when we have a little more time to think.

Reply:
Added an issue so we can remember to come back to this https://github.com/usds/case-issue-api/issues/40